libcontainer/README.md

```diff
@@ -317,7 +317,7 @@ state, err := container.State()
 #### Checkpoint & Restore
 
 libcontainer now integrates [CRIU](http://criu.org/) for checkpointing and restoring containers.
-This let's you save the state of a process running inside a container to disk, and then restore
+This let you save the state of a process running inside a container to disk, and then restore
```

Comment:
This doesn't look correct; I think the correct fix here would be
```suggestion
This lets you save the state of a process running inside a container to disk, and then restore
```

Reply:
Thanks for the correction, I modified it.